src/containers/Desk.js

```diff
@@ -3,56 +3,126 @@ import NavBar from '../components/NavBar'
 import Plane from '../components/Plane'
 import Read from '../components/Read'
 import Write from '../components/Write'
+import Journal from '../components/Journal'
+import CreateResponse from '../components/CreateResponse'
+
+const LETTERS_URL = 'http://localhost:3000/letters'
 
 class Desk extends Component {
   state = {
     letterStack: [],
-    logged_in: null
+    isWrite: false,
+    isRead: false,
+    isJournal: false,
+    plane: null
   }
 
   componentDidMount() {
-    // Fetch the letters and setState using PrevState to update
     this.fetchLetters();
   }
 
   fetchLetters = () => {
-    // Call this function on an interval to get more letters
-    // Ideally grab 10-20 at a time? Do it every 30 seconds?
-    const LETTERS_URL = 'http://localhost:3000/letters'
-
     fetch(LETTERS_URL)
       .then(resp => resp.json())
       .then(json => {
-        this.setState({ letterStack: json })
+        this.setState({ 
+          letterStack: json,
+          plane: json[Math.round(Math.random() * 10)] //this is just for testing!
+        })
       })
   }
 
+  handleWriteClick = () => {
+    this.setState({
+      isWrite: true
+    })
+  }
+
   renderWrite = () => {
-    console.log('write')
-    return <Write />
+    return (
+      <Write accountId={this.props.accountId} handleCloseClick={this.handleCloseClick} />
+    )
   }
 
-  renderRead = e => {
-    console.log(e.target.id, this)
-    return <Read />
+  handlePlaneClick = (_e, letter) => {
+    this.setState({
+      isRead: true
+    })
+    this.incrementLetterViews(letter)
   }
 
-  renderPlanes = () => {
-    return this.state.letterStack.map(letter => {
-      return <Plane key={letter.id} letter={letter} handleClick={this.renderRead} />
+  incrementLetterViews = (letter) => {
+    fetch(LETTERS_URL + `/${letter.id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+      },
+      body: JSON.stringify({
+        num_views: letter.num_views + 1 
+      })
     })
+    .then(res => res.json())
+    .then(console.log)
   }
 
+  renderRead = (letter) => {
+
+    return (
+      <Read letter={letter} handleWriteClick={this.handleWriteClick} />
+    )
+  }
+
+  renderCreateResponse = (letter) => {
+    return (
+      <CreateResponse accountId={this.props.accountId} letter={letter} isRead={true} isWrite={true} handleCloseClick={this.handleCloseClick}/>
```

Comment:
```suggestion
      <CreateResponse 
        accountId={this.props.accountId} 
        letter={letter} 
        isRead={true} 
        isWrite={true} 
        handleCloseClick={this.handleCloseClick} />
```

Reply:
done, updated Prettier which modified it